site/layouts/partials/scripts.html

```diff
@@ -4,15 +4,16 @@
   <script src="/docs/{{ .Site.Params.docs_version }}/dist/js/bootstrap.bundle.js"></script>
 {{- end }}
 
+<script src="https://cdn.jsdelivr.net/npm/clipboard@latest/dist/clipboard.min.js"></script>
```

Comment:
```suggestion
<script src="https://cdn.jsdelivr.net/npm/clipboard@2.0.11/dist/clipboard.min.js"></script>
```

Not sure if is better to use a specific version

Reply:
I think it's best to use a specific version for security reasons, yeah?

Reply:
Done ;)